extension/app/src/components/conversation/AttachFragment.tsx

```diff
@@ -32,57 +23,68 @@ export const AttachFragment = ({
     return () => clearTimeout(timer);
   }, [attachPageBlinking]);
 
-  const fileInputRef = useRef<HTMLInputElement>(null);
   return (
     <>
-      <input
-        accept={supportedFileExtensions.join(",")}
-        onChange={async (e) => {
-          await fileUploaderService.handleFileChange(e);
-          if (fileInputRef.current) {
-            fileInputRef.current.value = "";
+      <div className="block sm:hidden">
+        <Button
+          icon={DocumentPlusIcon}
+          tooltip="Extract text from page and attach"
+          variant="outline"
+          size="sm"
+          className={attachPageBlinking ? "animate-[bgblink_200ms_3]" : ""}
+          onClick={() =>
+            fileUploaderService.uploadContentTab({
+              includeContent: true,
+              includeCapture: false,
+            })
           }
-          editorService.focusEnd();
-        }}
-        ref={fileInputRef}
-        style={{ display: "none" }}
-        type="file"
-        multiple={true}
-      />
-      <Button
-        icon={AttachmentIcon}
-        tooltip="Attach file"
-        variant="ghost"
-        size="xs"
-        onClick={async () => {
-          fileInputRef.current?.click();
-        }}
-      />
-      <Button
-        icon={DocumentPlusIcon}
-        tooltip="Attach tab content"
-        variant="ghost"
-        size="xs"
-        className={attachPageBlinking ? "animate-[bgblink_200ms_3]" : ""}
-        onClick={() =>
-          fileUploaderService.uploadContentTab({
-            includeContent: true,
-            includeCapture: false,
-          })
-        }
-      />
-      <Button
-        icon={CameraIcon}
-        tooltip="Attach tab screenshot"
-        variant="ghost"
-        size="xs"
-        onClick={() =>
-          fileUploaderService.uploadContentTab({
-            includeContent: false,
-            includeCapture: true,
-          })
-        }
-      />
+        />
+      </div>
+      <div className="block sm:hidden">
```

Comment:
I need those buttons to be at the same level, not wrapped in a div otherwise the parent `space-x-` class won't be valid with the send button. 

I tried putting the class at the button level but this messes with the button so I went with the easiest!